### ISSUE TITLE
Loss.. should be specified as either training loss or validation loss

### ISSUE THREAD

**Issue by FlogramMatt** [huggingface/transformers#35494]
### System Info

Doesn't apply.. using runpod

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

As you fine tune a new model, it shows a loss curve and numbers as it runs.. is that training loss or validation loss? Should be labelled

If training loss.. how can I view validation loss? I set 'val size', it should exist somewhere

### Expected behavior

Should be labeled more clearly

**Comment by Rajatavaa** [comment]
Could you please elaborate more or provide a script to replicate this?

**Comment by FlogramMatt** [comment]
I did figure it out, it is training loss.

 I mean when fine tuning the model, you have a 'loss' graph in bottom right and same for outputs in terminal and it's not obvious that it's training loss, not validation loss.  Would be nice if labelled better.

**Comment by Rajatavaa** [comment]
I mean if your code doesn't specify to plot a graph then it will not show any graph.
And if you are plotting the graph then you can label it correctly
